types/load-google-maps-api/load-google-maps-api-tests.ts

```diff
@@ -2,7 +2,7 @@ import loadGoogleMapsApi = require('load-google-maps-api');
 
 // Example from README.md https://github.com/yuanqing/load-google-maps-api#usage
 loadGoogleMapsApi().then((googleMaps) => {
-    new googleMaps.Map(document.querySelector('.map'), {
```

Comment:
Is this a breaking change? If so you should probably bump the semver (plus the TS min version change makes for good reasoning IMO)

Reply:
In versions [3.39] (current), [3.38], and [3.37], Map always requires an element.

[3.39]: https://developers.google.com/maps/documentation/javascript/reference/map#Map.constructor
[3.38]: https://developers.google.com/maps/documentation/javascript/reference/3.38/map#Map.constructor
[3.37]: https://developers.google.com/maps/documentation/javascript/reference/3.37/map#Map.constructor

<img src="https://user-images.githubusercontent.com/10235949/70914951-a3a91c00-2021-11ea-9c4c-b72b98e8440f.png" width="459 " height="338.5" />